student-store-ui/src/components/App/App.jsx

```diff
@@ -90,17 +110,33 @@ export default function App() {
     }
   };
 
-  const handleOnCheckoutFormChange = (name, value) => {};
-
-  const handleOnSubmitCheckoutForm = () => {};
+  const handleOnCheckoutFormChange = (name, value) => {
+    name === "email" ? setEmail(value) : setName(value);
+  };
+  const handleOnSubmitCheckoutForm = async () => {
+    const body = {
+      order: {
+        user: { name: name, email: email },
+        shoppingCart: [...shoppingCart],
```

Comment:
```suggestion
        shoppingCart
```

Pretty sure this would be equivalent.